src/flag_parens.cpp

```diff
@@ -38,6 +38,8 @@ chunk_t *flag_parens(chunk_t *po, UINT64 flags, c_token_t opentype, c_token_t pa
       if (  flags != 0
          || (parent_all && parenttype != CT_NONE))
       {
+         // Issue #1734
```

Comment:
Should this still be here?

Reply:
This is the next problem! The opening brace is not flaged. We have to look why.